### ISSUE TITLE
Changes to scipy functions in scipy version 1.14.0

### ISSUE THREAD

**Issue by joshuarwood** [USRA-STI/gdt-core#49]
I ran into the following error when using astro-gdt in a fresh python3.10.5 virtual environment: 
```
ImportError: cannot import name 'trapz' from 'scipy.integrate'
```
It appears related to the removal of the 'trapz' function in favor of 'trapezoid' per the [Scipy 1.14.0 patch notes](https://docs.scipy.org/doc/scipy/release/1.14.0-notes.html)
* scipy.integrate.{simps,trapz,cumtrapz} have been removed in favour of simpson, trapezoid, and cumulative_trapezoid.

We'll need to update instances of trapz and include a scipy>=1.14.0 requirement in setup.py to fix this. I can probably do that on ~Friday July 5th.

**Comment by AdamGoldstein-USRA** [comment]
@joshuarwood this is a duplicate issue to #48.

**Comment by BillCleveland-USRA** [comment]
I'm addressing this with an update.

**Comment by BillCleveland-USRA** [comment]
This is being addressed in the "whc_api_updates" branch.

**Comment by AdamGoldstein-USRA** [comment]
Addressed in PR #56.
